### ISSUE TITLE
Detach 'Keep token' threads vault:admin even though server doesn't check it

### ISSUE THREAD

**Issue by unforced** [ParachuteComputer/paraclaw#59]
Follow-up nit from PR #57 (paraclaw#56) reviewer pass.

\`web/ui/src/routes/VaultDetail.tsx\` — \`DetachModal.detach()\` always passes \`authExtraScopes: [\\\`vault:\\\${vaultName}:admin\\\`]\` regardless of \`revokeToken\`. When \`revokeToken\` is **false** (the \"Keep token\" path), paraclaw doesn't call the vault, so the narrow scope is irrelevant — the only 403 source is the paraclaw-side \`claw:write\`-ish check, which the broad set already covers.

Result: harmless over-scoping. The current code works correctly; it just requests a scope it doesn't strictly need on the Keep-token path.

Fix would be:

\`\`\`ts
authExtraScopes: revokeToken ? [\\\`vault:\\\${vaultName}:admin\\\`] : undefined,
\`\`\`

The trade-off: branching adds complexity for a no-op savings (consent UI on the hub side handles \"already granted\" cleanly per cli#75). I lean toward closing this as wontfix unless the consent screen renders something confusing for the over-asked scope.

**Comment by unforced** [comment]
Addressed by PR #61 (merged 2026-04-30).
